server/docker-compose.yml.windows

```diff
@@ -0,0 +1,39 @@
+version: "3.7"
```

Comment:
Can you rename this file to `docker-compose.windows.yml` so we get editor syntax highlighting?